src/Resources/views/Download/creditMemo.html.twig

```diff
@@ -101,6 +101,18 @@
             </tr>
         {% endfor %}
 
+        <tr class="total">
+            <td colspan="6"></td>
+            <td>{{ 'sylius_refund.ui.net_total'|trans([], 'messages', creditMemo.localeCode) }}:</td>
+            <td>{{ '%0.2f'|format(creditMemo.getNetValueTotal()/100) }}</td>
+            <td>{{ creditMemo.currencyCode }}</td>
+        </tr>
+        <tr class="total">
+            <td colspan="6"></td>
+            <td>{{ 'sylius_refund.ui.tax_total'|trans([], 'messages', creditMemo.localeCode) }}:</td>
+            <td>{{ '%0.2f'|format(creditMemo.getTaxTotal()/100) }}</td>
+            <td>{{ creditMemo.currencyCode }}</td>
+        </tr>
```

Comment:
```suggestion
        <tr class="totalNet">
            <td colspan="6"></td>
            <td>{{ 'sylius_refund.ui.net_total'|trans([], 'messages', creditMemo.localeCode) }}:</td>
            <td>{{ '%0.2f'|format(creditMemo.getNetValueTotal()/100) }}</td>
            <td>{{ creditMemo.currencyCode }}</td>
        </tr>
        <tr class="totalTax">
            <td colspan="6"></td>
            <td>{{ 'sylius_refund.ui.tax_total'|trans([], 'messages', creditMemo.localeCode) }}:</td>
            <td>{{ '%0.2f'|format(creditMemo.getTaxTotal()/100) }}</td>
            <td>{{ creditMemo.currencyCode }}</td>
        </tr>
```

In case of CSS changes, this could be usefull ( so we can change only one table), wdyt?

Reply:
I have a totally opposite opinion, each row have the same class because they are styled in the same way, if someone will need to customize it, then he can add a new class or specific id.